src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMappingTest.java

```diff
@@ -948,7 +948,7 @@ void getIndividualRelatedParties() {
         assertThat(result)
             .isNotEmpty()
             .extracting("relatedPartyId", "relationshipType")
-            .contains(tuple("1", REPRESENTATIVE.getHmcReference()));
+            .contains(tuple("1", REPRESENTATIVE.getPartyRelationshipType().getRelationshipTypeCode()));
```

Comment:
This is a test so use SOLICITOR.getRelationshipTypeCode() as that is what we are testing and it makes the test clearer

Reply:
Replaced with SOLICITOR.getRelationshipTypeCode()